### ISSUE TITLE
Battery Charge

### ISSUE THREAD

**Issue by js2191001** [mooltipass/moolticute#1249]
### Expected behavior
Charge the device
### Actual behavior
Will not keep charge, after removal of power the device goes dead,

- The Mooltipass Mini BLE

**Comment by limpkin** [comment]
hello here!
Could you let us know the bundle version running on your device? did you perform a reconditioning procedure, documented in our user manual?
thanks!

**Comment by limpkin** [comment]
hello here!
Could you let us know the bundle version running on your device? did you perform a reconditioning procedure, documented in our user manual?
thanks!

**Comment by ananthb** [comment]
Hey @limpkin I'm running the latest bundle and I tried two reconditions. I believe that battery might be dead/dying on mine.

**Comment by limpkin** [comment]
what was the reported elapsed time after the procedure completed?
What's the story behind your unit? a bundle version 65535 is quite odd...

**Comment by js2191001** [comment]
Never was able to get it to hold a charge.

Joe Schwartz
630-219-1001 Cell

**Comment by limpkin** [comment]
right, but what was reported at the end of the reconditioning procedure?

**Comment by ananthb** [comment]
Yeah I'm pretty sure it originally displayed a different bundle version. The number is pretty suspect too. 

I ran the reconditioning process twice and I think it printed out a charge discharge duration. I remember some time interval being displayed.

Should I open a new issue for my bundle version?

**Comment by limpkin** [comment]
I might have sent you a beta firmware... I don't recall. Do you remember the time interval displayed?

**Comment by js2191001** [comment]
I'll have to check on Monday, I device is in my home office right now.
Joe Schwartz
630-219-1001 Cell

**Comment by ananthb** [comment]
It said 12 hours maybe. @limpkin would it be useful for me to run the conditioning again?

**Comment by limpkin** [comment]
@ananthb maybe yes :)

**Comment by ananthb** [comment]
The mooltipass' battery is stuck at 0%. I'm also not able to open the advanced tab, by right-clicking on the logo and by using the keyboard shortcut. Not really sure what to do now.

**Comment by js2191001** [comment]
still tried everything, can’t update it.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by limpkin** [comment]
@js2191001 can you let me know your unit's serial number?
